src/components/loginView.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {defineMessages, injectIntl, intlShape} from 'react-intl';
+import Dialog from 'material-ui/Dialog';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import TextField from 'material-ui/TextField';
+import FlatButton from 'material-ui/FlatButton';
+import Snackbar from 'material-ui/Snackbar';
+
+const messages = defineMessages({
+  title: {
+    id: 'loginview.title',
+    description: 'Modal Title',
+    defaultMessage: 'Login'
+  },
+  loginbutton: {
+    id: 'saveview.loginbutton',
+    description: 'Text for Login Button',
+    defaultMessage: 'Login'
+  },
+  closebutton: {
+    id: 'saveview.closebutton',
+    description: 'Text for Close Button',
+    defaultMessage: 'Close'
+  },
+  username: {
+    id: 'saveview.username',
+    description: 'Text for username label',
+    defaultMessage: 'Username'
+  },
+  password: {
+    id: 'saveview.password',
+    description: 'Text for Password Label',
+    defaultMessage: 'Password'
+  },
+  errormsg: {
+    id: 'saveview.errormsg',
+    description: 'Text for Save Error message',
+    defaultMessage: 'Error. {msg}'
+  },
+});
+
+export class LoginView extends React.Component {
```

Comment:
was there a way where we could use re-use some of the SDK login modal code, or was this not an option?